Library/Homebrew/dev-cmd/livecheck.rb

```diff
@@ -90,6 +92,31 @@ def run
           end
         end
 
+        if skip_autobump?
+          autobump_files = {}
```

Comment:
A small nit but `autobump_files` suggests to me that these are either `File` objects or the unparsed contents of the files. Since these are arrays of package names, something like `autobump_lists` or `autobump_packages` may be more in line. It could just be me, though.